### DISCUSSION TITLE
Librerias

### DISCUSSION THREAD

**Discussion by AnGys09** [INGCOM-UNRN-P1/cursada-2023#116] [Preguntas y respuestas] [answered]
podemos usar librerías adicionales, como <string.h> y otras???
 específicamente para el tp3

**Comment by inomdedeu** [comment]
Especialmente para el TP3, la idea es no usar librerias la libreria <string.h>
El TP3 esta pensado para que se familiaricen con la manipulación de cadenas y arreglos.
Muchos de los ejercicios son la implementación de funciones de esa libreria.

**Comment by martinvilu** [comment, marked as answer]
No porque la idea es implementar las funciones de dicha librería.

Y porque la librería en sí, no es segura.
